pipelines/oracle_grounded/distill_blocks.py

```
@@ -0,0 +1,298 @@
+
+
+def _measurements_shape_errors(block: dict[str, Any], status: str, where: str) -> list[str]:
+    """An array, and a non-empty one when the result claims to be measured."""
+
+    measurements_block = block.get("measurements")
+    if not isinstance(measurements_block, list):
+        return [f"{where}.result.measurements must be an array"]
+    if status == vocab.RESULT_MEASURED and not measurements_block:
+        return [
+            f"{where}.result: ORACLE_RESULT_MISSING — a measured result needs at "
+            "least one measurement"
```

Ruff (0.16.3)

[warning] Unparenthesized implicit string concatenation in collection

Did you forget a comma?

(ISC004)